.ci_support/linux_64_cuda_compilernvcccuda_compiler_version11.2cxx_compiler_version10python3.10.____cpython.yaml

```diff
@@ -1,34 +0,0 @@
-cdt_name:
-- cos7
-channel_sources:
-- conda-forge
-channel_targets:
-- conda-forge main
-cuda_compiler:
-- nvcc
-cuda_compiler_version:
-- '11.2'
```

Comment:
Earlier this year, [CUDA 11.2 was dropped in conda-forge]( https://conda-forge.org/news/2024/03/06/dropping-cuda-112/ ). Hence why this goes away in the re-render